extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -861,7 +861,7 @@ export class ProjectsController {
 		try {
 			const workspaceApi = utils.getDataWorkspaceExtensionApi();
 
-			const validateWorkspace = await workspaceApi.validateWorkspace();
+			const validateWorkspace = true; // await workspaceApi.validateWorkspace();
```

Comment:
should this be commented out?

Reply:
I ended up adding that back (had removed it earlier) so just reverting back to original code.